internal/test/wsu/wsu_test.go

```diff
@@ -100,15 +96,15 @@ func setupAndTest(t *testing.T, vmNum int) {
 	t.Parallel()
 
 	// Run the WSU against the VM
-	wsuOut, err := runWSU(framework.WinVMs[vmNum])
+	wsuOut, err := runWSU(framework.WinVMs[vmNum], vmNum)
```

Comment:
nit: Ideally this should be property of the Windows VM. Please think about adding 10 more VM, would numbering help then?

Reply:
I agree with this and would like to see this change. In a seperate commit of this PR you can add a getter to the interface which will return the purpose of the VM. This proposed commit would go before this one.

Reply:
Thank you for pointing it out. I have added a property to the interface